dist/tools/firmware_metadata/README.md

```diff
@@ -0,0 +1,33 @@
+# Metadata generator for OTA firmware updates
+This program will generate a binary file containing a metadata structure as
+follows:
+
+```c
+typedef struct FW_metadata {
```

Comment:
I didn't see the forward declaration? Anyhow, I mean, the uint32_t will be 32bit aligned on arm, but only 16bit aligned on e.g., msp430. So unless both defines used above them add up to multiples of 4 bytes, the struct alignment will change depending on the used platform. (might even change on 64bit host platforms)